hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AUtils.java

```diff
@@ -193,7 +193,7 @@ public static IOException translateException(@Nullable String operation,
         // call considered an sign of connectivity failure
         return (EOFException)new EOFException(message).initCause(exception);
       }
-      if (exception instanceof NoAuthWithAWSException) {
+      if (exception instanceof CredentialInitializationException) {
```

Comment:
can remove the now-unused import statement for NoAuthWithAWSException

Reply:
done